src/main/java/eu/execom/pomodoroTeam/controllers/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,33 @@
+package eu.execom.pomodoroTeam.controllers;
+
+import org.jboss.logging.Logger;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import javax.persistence.EntityNotFoundException;
+
+@RestControllerAdvice
+public class GlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    private static Logger log = Logger.getLogger(GlobalExceptionHandler.class);
+
+    @ExceptionHandler(EntityNotFoundException.class)
+    public ResponseEntity<Object> handleEntityNotFoundException(EntityNotFoundException e, WebRequest request) {
+        log.info("Entity not found!");
+        return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(Exception.class)
+    public ResponseEntity<Object> handleInternalServerErrorException(Exception e, WebRequest request) {
+        log.info("Internal server error!");
+        return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+
+    private void logException(Exception e) {
+        log.info("Exception" + e + "occurred!");
```

Comment:
This method is not used anywhere - either remove it, or use it. Also, to log exceptions, use error level, and logger (from SLF4j) provides a method to also print the stacktrace. E.g., 
```
log.error("Exception occured: ", e);
```